styles/_uswds-theme-custom-styles.scss

```diff
@@ -382,21 +400,44 @@ HOME
 .usa-search--domain_message-choosing {
   max-width: 400px;
 
-  .usa-alert--success.usa-alert--slim::before {
-    height: 3.5rem;
-  }
-
-  .usa-alert--error.usa-alert--slim::before {
-    height: 3.5rem;
-  }
-
   p:last-of-type {
     margin-bottom: 0.5rem;
   }
 
 }
 
+.usa-alert--success.usa-alert--slim::before {
+  height: 3.5rem;
+}
+
+.usa-alert--error.usa-alert--slim::before {
+  height: 3.5rem;
+}
+
+.usa-alert--info.usa-alert--slim::before {
+  height: 3.5rem;
+}
```

Comment:
Previously existing workaround for misalignment on USWDS icons on their alerts -- applied this to be global since we're adding more USWDS alerts in this PR